Code/GraphMol/MMPA/Wrap/testMMPA.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import print_function
```

Comment:
it's ok to have this here since it may help in the future, but it's not needed for this file at the moment.